models/cachetime.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "time"
+
+type CacheTime struct {
+	ID           string    `json:"_id" bson:"_id"`                   // MD5 of the path
+	Path         string    `json:"path" bson:"path"`                 // Path for which caching is set
+	ETag         string    `json:"etag" bson:"etag"`                 // ETag for cache validation
+	CollectionID int       `json:"collectionId" bson:"collectionId"` // Collection ID
```

Comment:
Given 'CollectionID' is a widely used piece of terminology that we refer to Zebedee collections - could you be more specific what this is being used for?